tests/AssertTest.php

```diff
@@ -289,8 +290,18 @@ public function getTests()
             array('uuid', array('ff6f8cb0-c57da-51e1-9b21-0800200c9a66'), false),
             array('uuid', array('af6f8cb-c57d-11e1-9b21-0800200c9a66'), false),
             array('uuid', array('3f6f8cb0-c57d-11e1-9b21-0800200c9a6'), false),
-
+            array('throws', array(function() { throw new LogicException('test'); }, 'LogicException'), true),
+            array('throws', array(function() { throw new LogicException('test'); }, 'IllogicException'), false),
+            array('throws', array(function() { throw new Exception('test'); }), true),
         );
+        if (interface_exists('Throwable')) {
```

Comment:
should we really? isn't it better to do "feature checking", instead of "version checking" ? (a la javascript :))